src/FormBuilder.php

```diff
@@ -136,13 +136,32 @@ private function addField(SymfonyFormBuilder $formBuilder, string $name, array $
             $attr = [];
 
             if ($this->hasRecaptchaV3) {
+                //Fix for allowing multiple recaptcha v3 forms on a single page
+
+                //Used to converted snake case into camel case
+                $splitFormName = explode("_", $formName);
+                if(count($splitFormName) > 1){
+                    foreach($splitFormName as $item){
+                        $item = ucfirst($item);
+                    }
+                    $formNameJs = join(" ", $splitFormName);
+                } else {
+                    $formNameJs = ucfirst($formName);
+                }
                 $attr = [
                     'class' => 'g-recaptcha',
                     'data-sitekey' => $config['recaptcha']['public_key'],
-                    'data-callback' => 'onRecaptchaSubmitted',
+                    'data-callback' => 'onRecaptchaSubmitted' . $formNameJs,
                     // pass the name of the form as the Action for reCAPTCHA v3
                     'data-action' => $formName,
-                ];
+                ];                
+                // $attr = [
```

Comment:
Hi @sJones1997 , sorry for the late response on this!

It looks good, can you remove those commented out lines? I can merge it in and tag a release then.